src/pkg/fault/fault.go

```diff
@@ -216,6 +259,27 @@ func (e *localBus) AddRecoverable(err error) {
 	e.bus.AddRecoverable(err)
 }
 
+// AddSkip appends a record of a Skipped item to the local bus.
+// Importantly, skipped items are not the same as recoverable
+// errors.  An item should only be skipped under the following
+// conditions.  All other cases should be handled as errors.
+// 1. The conditions for skipping the item are well-known and
+// well-documented.  End users need to be able to understand
+// both the conditions and identifications of skips.
```

Comment:
Perhaps, a statement like: `well-known` issues can be found in the /docs/Well-known.md` or something similar.

Reply:
I'll look into where we might want to document these cases.